src/ConfigurationFactory.php

```diff
@@ -103,6 +103,12 @@ protected function createWithConfig(ContainerInterface $container, string $confi
             $configuration->setEntityListenerResolver($config['entity_listener_resolver']);
         }
 
+        if (is_string($config['schema_assets_filter'])) {
+            $configuration->setSchemaAssetsFilter($container->get($config['schema_assets_filter']));
+        } elseif (is_callable($config['schema_assets_filter'])) {
```

Comment:
Fairly sure we should only allow for a service name to be used here.

Besides that: we need tests for this addition :)

Reply:
Hey @Ocramius,

i am using a closure in my local development configuration to skip some schemas (i.e. schemas from the postgis tiger geocoder) and do not really want to create an extra class for that small fix. So i think using a closure would be pretty nice for development purposes. 

Since there are no tests for the configuration factory i thougth i could skip the extra work :hankey:

Reply:
Put that closure in a service? :D

Reply:
Ok, you won ;-),

i updated the code and do not accept closures anymore. In my local environment i just moved the closure to the di container configuration :-D